sources/platform/actors/running/runs_and_builds.md

```diff
@@ -81,8 +81,8 @@ flowchart LR
 
 You can abort runs with the statuses **READY**, **RUNNING**, or **TIMING-OUT** in two ways:
 
-- **Immediately** - this is the default option.
-- **Gracefully** - the actor run receives a signal about aborting via the `aborting` and `persistState` events and is force-aborted after 30 seconds. This is helpful in cases where you plan to resurrect the run later because it saves the actor's state. When resurrected, the actor can restart where it left off.
+- **Immediately** - this is the default option. The actor process is killed immediately with no grace period.
+- **Gracefully** - the actor run receives a signal about aborting via the `aborting` event and is granted a 30-second window to finish in-progress tasks before getting aborted. This is helpful in cases where you plan to resurrect the run later because it gives the actor a chance to persist its state. When resurrected, the actor can restart where it left off.
```

Comment:
```suggestion
- **Gracefully** - the Actor run receives a signal about aborting via the `aborting` event and is granted a 30-second window to finish in-progress tasks before getting aborted. This is helpful in cases where you plan to resurrect the run later because it gives the Actor a chance to persist its state. When resurrected, the Actor can restart where it left off.
```